src/back/TicketManager/src/PaymentService/TicketManager.PaymentService.Domain/Payments/Payment.cs

```diff
@@ -0,0 +1,28 @@
+using TicketManager.PaymentService.Domain.Common;
+
+namespace TicketManager.PaymentService.Domain.Payments;
+
+public class Payment: IAggregateRoot<Guid>, IOptimisticConcurrent
+{
+    public Guid Id { get; private init; }
+    public PaymentStatus PaymentStatus { get; private set; }
+    public DateTime DateCreated { get; private set; }
```

Comment:
I guess that `DateCreated` can be init only.

Reply:
Addressed in `f5d6d56e`.